docker-compose.yml

```
@@ -15,6 +15,15 @@ services:
     command: server
     cap_add:
       - IPC_LOCK
+  vault-unsealer:
+    depends_on: [vault]
+    image: samply/bridgehead-vaultfetcher:latest
```

```suggestion
    image: samply/vaultfetcher:latest
```

This has been renamed